src/cow-react/utils/isCurrencyAmountFalsy.ts

```diff
@@ -0,0 +1,7 @@
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
+
+export function isCurrencyAmountFalsy(amount: CurrencyAmount<Currency> | null | undefined): boolean {
```

Comment:
great you included this. 

In fact, for a follow up it would be nice to allow also overload this function and allow `Fraction`

Then we can use it in many places :) 

i.e. 
* https://github.com/cowprotocol/cowswap/blob/408d298d0f6d20723911db12ea07566a5ffd440b/src/cow-react/modules/limitOrders/containers/RateInput/index.tsx#L31
* https://github.com/cowprotocol/cowswap/blob/408d298d0f6d20723911db12ea07566a5ffd440b/src/cow-react/modules/limitOrders/hooks/useApplyLimitRate.ts#L15
* https://github.com/cowprotocol/cowswap/blob/408d298d0f6d20723911db12ea07566a5ffd440b/src/cow-react/modules/limitOrders/hooks/useRateImpact.ts#L9
* https://github.com/cowprotocol/cowswap/blob/408d298d0f6d20723911db12ea07566a5ffd440b/src/cow-react/modules/limitOrders/updaters/MarketPriceUpdater/index.tsx#L16
* https://github.com/cowprotocol/cowswap/blob/408d298d0f6d20723911db12ea07566a5ffd440b/src/cow-react/pages/Account/Balances.tsx#L70
* ...